### PR TITLE
Release 0.1.3-alpha

### PR DESCRIPTION
**Pull request by devinreams**
@m8ttyB This PR is the next point release showing the latest changes on the default branch (master) compared to production.

Can you please help by re-testing and giving us your final approval to deploy to production here?

Per usual, we'll plan to address any concerns and aim for a push to production on Monday next week once we're all set. ✅

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "presets": [
     "react",
-    ["env", {"targets": {"browsers": ["firefox >= 54"]}}]
+    ["env", {"targets": {"browsers": ["firefox >= 56"]}}]
```
Comment (pdehaan) [inline .babelrc]: 56 or 57?

Comment (jimporter) [reply .babelrc]: 56, since Babel isn't happy with 57 (they haven't updated their env presets, at least not when this was committed).

**Comment by m8ttyB** [comment]
PI verified: manual regression [suites pass](https://testrail.stage.mozaws.net/index.php?/plans/view/7403), automated tests are passing, and door hanger work appears to be correctly pref'd off.

Thumbs up from the chaos cat
stella

**Comment by devinreams** [comment]
Thank you, @m8ttyB. ✅ 

I'll start the [release process](https://github.com/mozilla-lockbox/lockbox-extension/blob/master/docs/releases.md) in ~1 hour and report back here once live! 🚂 ⛵️

**Comment by devinreams** [comment]
OK, version `0.1.3-alpha` is in production and updated in Firefox as expected:

screen shot 2017-12-04 at 10 42 53 am

Here's the release on GitHub:

https://github.com/mozilla-lockbox/lockbox-extension/releases/tag/0.1.3-alpha

**Pull request merged**